packages/popover/package.json

```diff
@@ -24,6 +24,7 @@
     "@leafygreen-ui/hooks": "^6.0.1",
     "@leafygreen-ui/lib": "^6.1.1",
     "@leafygreen-ui/portal": "^3.0.1",
+    "@leafygreen-ui/leafygreen-provider": "^2.0.2",
```

Comment:
should this be a peerDep?

Reply:
Nope – we consume a hook from that package.